packages/backend/src/open_payments/payment/combined/service.ts

```diff
@@ -0,0 +1,58 @@
+import { TransactionOrKnex } from 'objection'
+import { BaseService } from '../../../shared/baseService'
+import { Pagination } from '../../../shared/baseModel'
+import { CombinedPayment } from './model'
+import { FilterString } from '../../../shared/filters'
+import { OutgoingPaymentService } from '../outgoing/service'
+import { IncomingPaymentService } from '../incoming/service'
+
+interface CombinedPaymentFilter {
+  paymentPointerId?: FilterString
+  type?: FilterString
+}
+interface GetPageOptions {
+  pagination?: Pagination
+  filter?: CombinedPaymentFilter
+}
+
+export interface CombinedPaymentService {
```

Comment:
We should add this to the `AppServices` interface so it's typed when it's used

Reply:
fixed, thank you